### PR TITLE
a few improvements

### PR DESCRIPTION
**Pull request by letzya**
Added explanations 
Removed "Default" from versin_data as it's not a type

Signed-off-by: Yaara <3155222+letzya@users.noreply.github.com>

### CODE REVIEW

File: JSON/draft-07/schema_apidef_lean.json [hunk 1]
```diff
@@ -177,53 +178,26 @@
...
-                        }
                     },
                     "additionalProperties": true
                 },
                 "not_versioned": {
                     "$id": "#/properties/version_data/properties/not_versioned",
                     "type": "boolean",
                     "title": "The not_versioned schema",
-                    "description": "An explanation about the purpose of this instance.",
+                    "description": "Define if the api has versioning turned on. True by default, i.e. not versioned.",
```
Comment (buraksekili) [inline JSON/draft-07/schema_apidef_lean.json:190]: Is version_data.not_versioned True by default? As far as i understand from this documentation https://tyk.io/docs/tyk-apis/tyk-gateway-api/api-definition-objects/versioning-endpoint/#versioning-in-the-url-path, the default value of  `version_data.not_versioned` is False.

> version_data.not_versioned: If set to true Tyk will skip version checking, you can still block or allow information to your API by specifying a Default version within the versions map.

```suggestion
                    "description": "Define if the api has versioning turned on. False by default, i.e. not versioned.",
```

Comment (buraksekili) [inline JSON/draft-07/schema_apidef_lean.json:190]: Is version_data.not_versioned True by default? As far as i understand from this documentation https://tyk.io/docs/tyk-apis/tyk-gateway-api/api-definition-objects/versioning-endpoint/#versioning-in-the-url-path, the default value of  `version_data.not_versioned` is False.

> version_data.not_versioned: If set to true Tyk will skip version checking, you can still block or allow information to your API by specifying a Default version within the versions map.

```suggestion
                    "description": "Define if the api has versioning turned on. False by default, i.e. not versioned.",
```

Comment (letzya) [reply JSON/draft-07/schema_apidef_lean.json]: @buraksekili False for `not_versioned` means it is versioned

**Review by buraksekili — APPROVED** [review]
thank you @letzya

**Pull request merged**